sklearn/svm/_base.py

```diff
@@ -861,11 +861,10 @@ def _fit_liblinear(X, y, C, fit_intercept, intercept_scaling, class_weight,
         Stopping condition.
 
     random_state : int or RandomState instance, default=None
-        The seed of the pseudo random number generator to use when shuffling
-        the data.  If int, random_state is the seed used by the random number
-        generator; If RandomState instance, random_state is the random number
-        generator; If None, the random number generator is the RandomState
-        instance used by `np.random`.
+        Controls the pseudo random number generator to use when shuffling the
```

Comment:
I think this phrasing is confusing. To me it means that one can choose the random number generator. But actually with numpy random state there's only one generator (unlike with numpy's new Generator API which we don't use yet). So I'd say something like  "Controls the pseudo random number generation for shuffling the data"

Reply:
I addressed your comment @jeremiedbb